### DISCUSSION TITLE
Welcome to cimgui-go Discussions!

### DISCUSSION THREAD

**Discussion by AllenDang** [AllenDang/cimgui-go#47] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by runrc** [comment]
@AllenDang (and others!) Thank you for providing cimgui-go. 

Here are some Dear ImGui resources that I have found very useful in learning Dear ImGui ...

[Dear ImGui manual ](https://pthom.github.io/imgui_manual_online/manual/imgui_manual.html) A useful online visual manual for the original  C++ implementation of Dear ImGui. It is easy and straightforward enough to translate to corresponding cimgui-go values and functions.

[Dear ImGui docking demo](https://traineq.org/HelloImGui/hello_imgui_demos/hello_imgui_demodocking/hello_imgui_demodocking.html) Useful Dear ImGui demo highlighting useful docking and viewport features.

[Dear ImGui Wiki](https://github.com/ocornut/imgui/wiki) A great starting point to understand some of the core concepts behind Dear ImGui.

[ImPlot demo](https://traineq.org/implot_demo/src/implot_demo.html) A demo of ImPlot.
